audb/core/publish.py

```diff
@@ -221,6 +222,8 @@ def publish(
             or ``None``
             if no version was published.
             If ``None`` it assumes you start from scratch.
+        cache_root: cache folder where databases are stored.
+            If not set :meth:`audb.default_cache_root` is used
```

Comment:
Should we maybe elaborate that we only store the header there?
Otherwise a user could assume that the database is not only published, but also moved into the cache.

Reply:
maybe:

![image](https://user-images.githubusercontent.com/10383417/114551623-f1f51c00-9c63-11eb-91ec-6f17f02f84df.png)

Reply:
Yes, that would be sufficient.

Reply:
done